docs/UserGuide.md

```diff
@@ -78,16 +81,44 @@ quickly get started and make full of use of what the application has to offer.
 
     * **`fav`** `2` : Adds the 2nd contact shown in the current list to your list of favourite contacts
 
-    * **`favourites`** : Lists all your favourite contacts.
-
-    * **`clear`** : Deletes all contacts.
+    * **`view`** `2` : Brings the 2nd contact into detailed view.
 
     * **`exit`** : Exits the app.
 
 6. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Tutorial
+
+1. This is the **list view**, the primary view you will work with when using the app.
+![Default list view](images/Sample1.png)
+2. In list view, all commands requiring an `INDEX` will reference the index provided on the left side of the contact's
+name. For example, typing the command `fav 2` will favourite the 2nd contact in the list.
+![Tutorial fav](images/TutorialFav.png)
+
+3. Add a new contact by running `add n/<YOUR NAME> p/<YOUR PHONE NUMBER> e/<YOUR EMAIL ADDRESS>`. These fields are 
+mandatory when creating a new contact. However, you can add more optional information in the same add command, or in a
+future edit command. See [adding a contact](#adding-a-contact--add) and [editing a contact](#editing-a-contact--edit)
+for more information.
+![Tutorial add](images/TutorialAdd.png)
+
+4. To bring the second contact into **detailed view**, input `view 2`.
+![Tutorial view](images/TutorialView.png)
+
+5. In **detailed view**, some commands available to you in **list view** are no longer available, however, your commands
+will now automatically reference the current contact you are viewing in **detailed view**. For example, you can type
+`impt` to simply label the current contact you are viewing to set them as an important contact.
+![Tutorial impt](images/TutorialImpt.png)
+
+6. To exit **detailed view**, simply input `list` to return to the **list view**.
+
+These are just the basic features to get you ready to start using the app. There are many more features for you to
+explore, it is recommended for you to have a read through our User Guide to familiarize yourself with the functionalities
+of the app. You can always input the `help` command whenever you need help or want to reference the User Guide again!
```

Comment:
I thought this was a great section added into the UG to help users get started! Nice work!